Source/gg2/Objects/Menus/Main Menu Elements/CreditsController.events/Mouse wheel down.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<event category="MOUSE" id="61">
+  <actions>
+    <action id="603" library="1">
+      <!--action name: Code-->
+      <kind>CODE</kind>
+      <allowRelative>false</allowRelative>
+      <question>false</question>
+      <canApplyTo>true</canApplyTo>
+      <actionType>CODE</actionType>
+      <functionName/>
+      <relative>false</relative>
+      <not>false</not>
+      <appliesTo>.self</appliesTo>
+      <arguments>
+        <argument kind="STRING">if (y - 30 * global.delta_factor &gt;= -1125) {
+    y -= 30 * global.delta_factor;
```

Comment:
Why with the delta factor? Isn't the mousewheel impulse-based, i.e. every activation should move by a fixed distance?

Reply:
You're right, it should,I'll fix it asap